setup.py

```diff
@@ -92,30 +80,28 @@ def run_tests(self):
     version=version,
     description=__doc__,
     long_description=readme + '\n\n' + history,
-    keywords='invenio TODO',
+    keywords='invenio minter pidstore autoincrement',
     license='GPLv2',
     author='CERN',
     author_email='info@inveniosoftware.org',
     url='https://github.com/inveniosoftware/invenio-sequencegenerator',
-    packages=[
-        'invenio_sequencegenerator',
-    ],
+    packages=packages,
     zip_safe=False,
     include_package_data=True,
     platforms='any',
     entry_points={
-        'console_scripts': [
-            'bibtex = invenio_sequencegenerator.scripts.bibtex:main',
+        'invenio_i18n.translations': [
+            'messages = invenio_sequencegenerator',
         ],
-    },
-    install_requires=requirements,
-    extras_require={
-        'docs': [
-            'Sphinx>=1.3',
-            'sphinx_rtd_theme>=0.1.7'
+        'invenio_db.models': [
```

Comment:
Did you remove the `models.py` file? If it's coming in next PR then please comment or remove this entrypoint. Thanks